network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/ConnectablePositionImpl.java

```diff
@@ -39,27 +39,32 @@ public Optional<String> getName() {
         }
 
         @Override
-        public Feeder setName(String name) {
+        public Feeder setName(String name, int side) {
```

Comment:
You can't add side here, because setName overrides the Feeder interface method in powsybl-core, which have only one argument ...
(same remark for setOrder and setDirection)

I think the solution is to add an new argument 'side' to the ConnectablePositionImpl constructor, and to change the 4 positionAttributes getter functions arguments, as something like that : 

BiFunction<Connectable<C>, ThreeSides, ConnectablePositionAttributes> positionAttributesGetter,